### PR TITLE
Add app router example

### PR DESCRIPTION
**Pull request by ghost**
- Add app router example to 03-environment-variables.mdx

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: d3992234f6e6f6ae751fe05dbfeb4ec98a2ad8d0

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/01-building-your-application/06-configuring/03-environment-variables.mdx [hunk 1]
```diff
@@ -48,9 +48,20 @@ export async function getStaticProps() {
 
 <AppOnly>
 
-This loads `process.env.DB_HOST`, `process.env.DB_USER`, and `process.env.DB_PASS` into the Node.js environment automatically allowing you to use them in Route Handlers.
+This loads `process.env.DB_HOST`, `process.env.DB_USER`, and `process.env.DB_PASS` into the Node.js environment automatically allowing you to use them in [Route Handlers](docs/app/building-your-application/routing/router-handlers).
 
-{/* TODO: App Router Example */}
+For Example:
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/06-configuring/03-environment-variables.mdx:53]:
```suggestion
For example:
```

**Review by delbaoliveira — CHANGES_REQUESTED** [review]

**Comment by delbaoliveira** [comment]
Thank you!

**Review by delbaoliveira — DISMISSED** [review]

File: docs/02-app/01-building-your-application/06-configuring/03-environment-variables.mdx [hunk 2]
```diff
@@ -48,9 +48,20 @@ export async function getStaticProps() {
 
 <AppOnly>
 
-This loads `process.env.DB_HOST`, `process.env.DB_USER`, and `process.env.DB_PASS` into the Node.js environment automatically allowing you to use them in Route Handlers.
+This loads `process.env.DB_HOST`, `process.env.DB_USER`, and `process.env.DB_PASS` into the Node.js environment automatically allowing you to use them in [Route Handlers](docs/app/building-your-application/routing/router-handlers).
```
Comment (styfle) [inline docs/02-app/01-building-your-application/06-configuring/03-environment-variables.mdx:51]:
```suggestion
This loads `process.env.DB_HOST`, `process.env.DB_USER`, and `process.env.DB_PASS` into the Node.js environment automatically allowing you to use them in [Route Handlers](/docs/app/building-your-application/routing/router-handlers).
```

**Review by styfle — APPROVED** [review]

**Pull request merged**